storage/engine/rocksdb/db.cc

```diff
@@ -1077,9 +1077,15 @@ class BaseDeltaIterator : public rocksdb::Iterator {
 }  // namespace
 
 DBStatus DBOpen(DBEngine **db, DBSlice dir, DBOptions db_opts) {
+  // Divide the cache space into two levels: the fast row cache
+  // and the slower but more space-efficient block cache.
+  // TODO(bdarnell): do we need separate settings for these?
+  const auto row_cache_size = db_opts.cache_size / 2;
+  const auto block_cache_size = db_opts.cache_size / 2;
```

Comment:
Perhaps `db_opts.cache_size - row_cache_size`. Did you do any experimentation of different values here?